### PR TITLE
Added Rating Component

### PR DESCRIPTION
**Pull request by anamkaa**
Added rating component
- rating icon
- rating badge with various color indications

### CODE REVIEW

File: comp-layouts/rating.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    <script
+      src="https://kit.fontawesome.com/c4d0bd5372.js"
+      crossorigin="anonymous"
+    ></script>
+    <script src="https://unpkg.com/phosphor-icons"></script>
+    <link rel="stylesheet" href="/stylesheets/main.css" />
+    <link rel="stylesheet" href="/stylesheets/components/rating.css" />
+</head>
+<body>
+    <div class="rating-cont">
```
Comment (dev-enforced) [inline comp-layouts/rating.html:17]: Can rename the class as rating-container for better understanding

File: docs/docs-rating.html [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+                </div>
+                <div class="nav-icon-chip">
+                    <a href="/docs/doc.html">
+                        <div class="nav-head nav-doc-head">
+                            <i class="fa-brands fa-github fa nav-icon btn-doc"></i></div>
+                    </a>
+                </div>
+            </div>
+        </nav>
+        <!--  -->
```
Comment (dev-enforced) [inline docs/docs-rating.html:41]: Can remove this comment

**Review by dev-enforced — APPROVED** [review]
The code structure looks good and the live preview of the ratings component is also well documented and easy to use: https://deploy-preview-41--fab-ui-lib.netlify.app/docs/docs-rating.html.Gave some suggestions below and would advise to use utility classes for margins and flex related properties to simplify and trim your CSS code

File: comp-layouts/rating.html (hunk 1 above)
Comment (anamkaa) [reply comp-layouts/rating.html:17]: okay

File: docs/docs-rating.html (hunk 2 above)
Comment (anamkaa) [reply docs/docs-rating.html:41]: Will rectify it

**Review by hrshmistry — COMMENTED** [review]
Rating component looks good 👍🏻

**Pull request merged**